internal/grpc/grpclog/grpclog_test.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+// grpclog provides a logging utilities for gRPC.
+package grpclog
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/connectivity"
+)
+
+func TestLogStateChanges(t *testing.T) {
+	testCases := map[string]struct {
+		name   string
+		conn   getStater
+		assert func(t *testing.T, lg *spyLog, isReadyCallbackCalled bool)
+	}{
+		"state: connecting, ready": {
+			conn: &stubConn{
+				states: []connectivity.State{
+					connectivity.Connecting,
+					connectivity.Ready,
+					connectivity.Ready,
+				},
+			},
+			assert: func(t *testing.T, lg *spyLog, isReadyCallbackCalled bool) {
+				require.Len(t, lg.msgs, 3)
+				assert.Equal(t, "Connection state started as CONNECTING", lg.msgs[0])
+				assert.Equal(t, "Connection state changed to CONNECTING", lg.msgs[1])
+				assert.Equal(t, "Connection ready", lg.msgs[2])
+			},
+		},
+		"state: ready": {
+			conn: &stubConn{
+				states: []connectivity.State{
+					connectivity.Ready,
+					connectivity.Idle,
+				},
+				stopWaitForChange: false,
+			},
+			assert: func(t *testing.T, lg *spyLog, isReadyCallbackCalledCallback bool) {
```

Comment:
```suggestion
			assert: func(t *testing.T, lg *spyLog, isReadyCallbackCalled bool) {
```
Please unify the naming.